openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1065,70 +1096,91 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
+  * `notification_id`: OPTIONAL. String identifying a batch of issued Credentials that the Wallet includes in the Notification Request as defined in (#notification). This parameter MUST NOT be present if `credential` parameter is not present.
+  * `transaction_id`: REQUIRED if `credentials` is not present within a particular Batch Credential Response object. It contains a String identifying a Deferred Issuance transaction. This claim is contained in the response if the Credential Issuer was unable to immediately issue the Credential. The value is subsequently used to obtain the respective Credential with the Deferred Credential Endpoint (see (#deferred-credential-issuance)). It MUST be invalidated after the Credential for which it was meant has been obtained by the Wallet.
+* `c_nonce`: OPTIONAL. The `c_nonce` as defined in (#credential-response).
+* `c_nonce_expires_in`: OPTIONAL. The `c_nonce_expires_in` as defined in (#credential-response).
 
-{
-  "credential_responses": [{
-    "credential": "eyJraWQiOiJkaWQ6ZXhhbXBsZTpl...C_aZKPxgihac0aW9EkL1nOzM"
-  },
-  {
-    "credential": "YXNkZnNhZGZkamZqZGFza23....29tZTIzMjMyMzIzMjMy"
-  }],
-  "c_nonce": "fGFF7UkhLa",
-  "c_nonce_expires_in": 86400
-}
-```
+The encoding of the Credential returned in the `credential` parameter depends on the Credential Format. Credential formats expressed as binary data MUST be base64url-encoded and returned as a string.
+
+More details such as Credential Format identifiers are defined in the Credential Format Profiles in (#format-profiles).
 
-Below is a non-normative example of a Batch Credential Response that contains Deferred Credential Response and Credential Response objects:
+Below is a non-normative example of a Batch Credential Response containing:
+* 2 Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys
+* 1 Credential for a different  Credential Configuration and Credential Dataset
+* 1 Transaction ID for a request that could not be immediately fulfilled
 
 ```
 HTTP/1.1 200 OK
 Content-Type: application/json
 Cache-Control: no-store
 
 {
-   "credential_responses":[
-      {
-         "transaction_id":"8xLOxBtZp8"
-      },
-      {
-         "credential":"YXNkZnNhZGZkamZqZGFza23....29tZTIzMjMyMzIzMjMy"
-      }
-   ],
-   "c_nonce":"fGFF7UkhLa",
-   "c_nonce_expires_in":86400
+  "credential_responses": [
+     {
+        "credentials": [
+           "eyJraWQiOiJkaWQ6ZXhhbXBsZTpl...C_aZKPxgihac0aW9EkL1nOzM",
+           "YXNkZnNhZGZkamZqZGFza23....29tZTIzMjMyMzIzMjMy"
+        ]
+     },
+     {
+        "credentials": [
+           "YXNkZnNhZGZkamZqZGFza23....29tZTIzMjMyMzIzMjMy"
+        ],
+        "notification_id" : "3fwe98js"
+     },
+     {
+        "transaction_id":"8xLOxBtZp8"
+     }
+  ],
+  "c_nonce": "fGFF7UkhLa",
```

Comment:
`c_nonce` seems like it should be more verbose here... does this stand for `credential_nonce`  or `challenge_nonce` ?

Reply:
From the definition: "c_nonce: OPTIONAL. String containing a nonce to be used to create a proof of possession of key material when requesting a Credential"
So I believe it stands for Credential